docs/versioned_docs/version-6.1.x/configuration/overview.md

```diff
@@ -82,7 +83,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--profile-url` | string | Profile access endpoint | |
 | `--prompt` | string | [OIDC prompt](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest); if present, `approval-prompt` is ignored | `""` |
 | `--provider` | string | OAuth provider | google |
-| `--provider-ca-file` |  string \| list |  Paths to CA certificates that should be used when connecting to the provider.  If not specified, the default Go trust sources are used instead. |
+| `--provider-ca-files` |  string \| list |  Paths to CA certificates that should be used when connecting to the provider.  If not specified, the default Go trust sources are used instead. |
```

Comment:
FYI, command line flags are singular and intended to be passed multiple times if you have multiples.

It is plural form in configuration files or environment variables.

Reply:
Thank you for pointing this out. I've updated the Pull Request Issue title and description details. 

As I've been working with Keycloak and found it's Auth Provider configuration description very limited in order to successfully startup OAuth2 Proxy with Keycloak, I'm proposing some changes also in the Auth.md (as mentioned in the details above)